apps/aeprimop/src/aeprimop.erl

```diff
@@ -1647,6 +1698,25 @@ assert_not_name(NameHash, S) ->
         none   -> ok
     end.
 
+assert_not_name_auction(AuctionHash, S) ->
+    case find_name_auction(AuctionHash, S) of
+        {_, _} -> runtime_error(name_already_in_auction);
+        none   -> ok
+    end.
+
+assert_valid_overbid(Protocol, NameAscii, NewNameFee, OldNameFee) ->
+    MinNameFee = aec_governance:name_claim_fee(NameAscii, Protocol),
+    Progression = aec_governance:name_claim_bid_increment(),
+    %% Stay within integer computations New bid must be Progression % higher
+    case (NewNameFee - OldNameFee) * 100 >= MinNameFee * Progression of
```

Comment:
I think we should not include MinNameFee here. For popular domains, spam incentive will be the highest and MinNameFee the lowest in comparison to {New,Old}NameFee.

Reply:
Indeed, now we compare to OldNameFee instead